docs/data/migration/migration-pickers-lab/migration-pickers-lab.md

```diff
@@ -94,11 +94,11 @@ The `disableCloseOnSelect` prop has been replaced by a new `closeOnSelect` prop
 The default behavior remains the same (close after the last step on desktop but not on mobile).
 
 ```diff
-// If you don't want to close after the last step
+ // If you don't want to close after the last step
 -<DatePicker disableCloseOnSelect={false} />
 +<DatePicker closeOnSelect />
 
-// If you want to close after the last step
+ // If you want to close after the last step
 -<DatePicker disableCloseOnSelect />
 +<DatePicker closeOnSelect={false} />
```

Comment:
IMHO—this is an awkward change, was it produced automatically by some sort of tool?

Reply:
It's the output `git diff` generates

Reply:
I'm sorry, I put the comment on the wrong line... I intended to comment the change of space before the comment on L101. That is what actually confused me and seemed strange—especially when we usually don't see such code formatting in actual code. 🤔

Reply:
Oh I see,

**Before**
<img width="525" alt="Screenshot 2022-10-17 at 13 27 54" src="https://user-images.githubusercontent.com/3165635/196165815-e074711b-018f-432e-8cb8-f1f066197375.png">
<img width="842" alt="Screenshot 2022-10-17 at 13 26 09" src="https://user-images.githubusercontent.com/3165635/196165839-eb8dcb7c-85a1-45c6-bc0f-265a74f3273b.png">

**After**
<img width="507" alt="Screenshot 2022-10-17 at 13 27 50" src="https://user-images.githubusercontent.com/3165635/196165808-0358bae4-e08e-4b3c-a126-4d985139b389.png">
<img width="852" alt="Screenshot 2022-10-17 at 13 25 37" src="https://user-images.githubusercontent.com/3165635/196165835-5baac84c-0fc1-422b-a847-807dab16e0f2.png">

Reply:
🤦 Sorry for wasting your time on this one...

Reply:
@LukasTy Don't be, we can never doubt enough about everything we see.